babel.config.js

```diff
@@ -11,11 +11,25 @@ function resolvePath(sourcePath, currentFile, opts) {
 
 let defaultPresets;
 
-// We release a ES version of Material-UI.
-// It's something that matches the latest official supported features of JavaScript.
-// Nothing more (stage-1, etc), nothing less (require, etc).
+// evergreen build
+// see https://next.material-ui.com/guides/minimizing-bundle-size/#evergreen-build
 if (process.env.BABEL_ENV === 'es') {
-  defaultPresets = [];
+  defaultPresets = [
+    [
+      '@babel/preset-env',
+      {
+        modules: false,
+        // snapshot of the latest version of evergreen browsers
+        targets: {
+          chrome: 73,
```

Comment:
I'm wondering, what happens if we set a version in the future like `chrome: 200`? Would it allow us not to have to maintain the version?

Reply:
The problem is that this would result in non-reproducible builds between browser releases.

We can document that our `/es` build always targets the last 1 version of evergreen browsers. This means that updating those versions could be done in a feature release. 

It seems like this would be the less painful strategy instead of random target changes between any release.

Reply:
Last 1 version sounds perfect to me for the problem we want to solve. I have seen them uisng our /es folder to apply another layer of Babel preset with their own targets.

Reply:
What I meant was a snapshot of "last 1 version". The query only makes sense for application code. Library code should use deterministic targets.